### DISCUSSION TITLE
Windows Terminal v1.16.1026 (1.16.10261.0 and 1.16.10262.0)

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#14749] [Announcements]
This release of Windows Terminal addresses a crash in self-elevation.

Huge thanks to @jboelter for fixing it in 1.17 (#14637).

---

### Asset Hashes

- Microsoft.WindowsTerminal_Win11_1.16.10262.0_8wekyb3d8bbwe.msixbundle
   - SHA256 `229ABC77AE04FC47037F1050C6971E967C840B368CFB92468A2E0EAA78245501`
- Microsoft.WindowsTerminal_Win11_1.16.10262.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip
   - SHA256 `17C03963403EF72244E85AD097022B109D9A8502A73F76C8D034DAEDB9D123C4`
- Microsoft.WindowsTerminal_Win10_1.16.10261.0_8wekyb3d8bbwe.msixbundle
   - SHA256 `BA6FC6854E713094B4009CF2021E8B4887CFF737AB4B9C4F9390462DD2708298`
- Microsoft.WindowsTerminal_Win10_1.16.10261.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip
   - SHA256 `14A7C5D6743D0FF22397B9CFA43576F7193A29F1727C04C49652CDCBFA168634`

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.16.10261.0'>Windows Terminal v1.16.1026 (1.16.10261.0 and 1.16.10262.0)</a>.</em>

**Comment by beingmerry** [comment]
Nice! Thought I was going crazy this morning when my terminal kept on crashing to desktop

**Reply by WoogonShim** [reply]
I still have the problem that the elevate-shim process restarts infinitely. :^(

I'm using Windows 10 Enterprise, 21H2 `19044.2006` build (Windows Feature Experience Pack 120.2212.4180.0)

**Reply by jboelter** [reply]
@WoogonShim You may be seeing this issue #13928

**Reply by WoogonShim** [reply]
Thank you, @jboelter,
I read your link https://github.com/microsoft/terminal/issues/13928#issuecomment-1374957536. I should use the `Administrator` user **only** because my laptop is managed by our company.

I just removed `elevate` setting from `pwsh` command and/or defaults setting. The problem disappeared!
```bash
    "profiles": 
    {
        "defaults": { "elevate": true },
    ...
```

Thank you again! :)

**Comment by pizzacrystal** [comment]
Sorry for bothering, but I'd like to ask how long will it take for pushing this stable version (for Win10) onto Microsoft Store so that it could be automatically updated?

**Reply by DHowett** [reply]
We're hoping to get it into GA by the end of the week. See my other update below (in a separate fork of this discussion) for more info on our rollout. :)

**Comment by DHowett** [comment]
I just pressed [Go] on the rollout to Beta and Release Preview. Early metrics suggest that we're safe to roll out more broadly, with roughly the same crash rate in the wild as the last stable release, v1.15.346x.0.

**Comment by John0King** [comment]
May I ask that when can I update from ms store ?

**Reply by zadjii-msft** [reply]
From the FAQs: [Version XYZ was just released, why doesn't it show up as an update in the Store?
](https://github.com/microsoft/terminal/wiki/Frequently-Asked-Questions-(FAQ)#version-xyz-was-just-released-why-doesnt-it-show-up-as-an-update-in-the-store)

**Comment by DHowett** [comment]
This is now rolling out to the rest of the world. Woo!

**Comment by rvnexe** [comment]
Add-AppxPackage : Deployment failed with HRESULT: 0x80073CFD, A Prerequisite for an install could not be satisfied.
Windows cannot install package Microsoft.WindowsTerminal_1.16.10262.0_x64__8wekyb3d8bbwe because this package is not
compatible with the device. The package requires OS version 10.0.22000.0 or higher on the Windows.Desktop device
family. The device is currently running OS version 10.0.19043.1237.

Can I fix this or There's a previous version that supports my OS?

**Reply by zadjii-msft** [reply]
@aL1NaZ4R That looks like you tried to install the Microsoft.WindowsTerminal_**Win11**_1.16.10262.0_8wekyb3d8bbwe.msixbundle bundle. You'll need the other one, the Win10_1.16.10261.0 file 😉

**Reply by rvnexe** [reply]
Ow, i'll try it. Thanks ^w^

**Comment by rvnexe** [comment]
how I can fix this?
![image](https://user-images.githubusercontent.com/89677246/224490989-229e94bd-31c8-4e5a-9a52-735853e039e5.png)
